lib/train/transports/local.rb

```diff
@@ -52,6 +59,154 @@ def login_command
       def uri
         'local://'
       end
+
+      class GenericRunner
+        def run_command(cmd)
+          res = Mixlib::ShellOut.new(cmd)
+          res.run_command
+          Local::CommandResult.new(res.stdout, res.stderr, res.exitstatus)
+        end
+      end
+
+      class WindowsRunner
```

Comment:
if we move the creation to the initialize phase of `Connection`, I think we do not need the `WindowsRunner` abstraction.